app/views/dashboard/index.html.erb

```diff
@@ -0,0 +1,62 @@
+<h1>Dashboard</h1>
+<table border="1">
+  <thead>
+    <th>Tenant</th>
+    <th>API Key</th>
+    <th>API Request Count</th>
+  </thead>
+  <tbody>
+    <% @tenants.each do |tenant|%>
+      <tr>
+        <td><%= tenant.name%></td>
+        <td><%= tenant.api_key %></td>
+        <td><%= tenant.tenant_api_requests.map(&:count).sum %></td>
+      </tr>
+    <% end %>
+  </tbody>
+</table>
+<br/>
+<table border="1">
+  <thead>
+    <th>User</th>
+    <th>Questions</th>
+    <th>Answers</th>
+  </thead>
+  <tbody>
+    <% @users.each do |user| %>
+      <tr>
+        <td>
+          <%= user.name %>
+        </td>
+        <td>
+          <table>
+            <thead></thead>
+            <tbody>
+              <% user.questions.each_with_index do |question, index| %>
+                <tr>
+                  <td>
+                    <%= index + 1 %>. <%= question.title %> <b><%= question.private ? '(Private)' : ''%></b>
+                  </td>
+                </tr>
+              <% end %>
+            </tbody>
+          </table>
+        </td>
+        <td>
+          <table>
+            <thead></thead>
+            <tbody>
+              <% user.answers.each_with_index do |answer, index| %>
+                <tr>
+                  <td>
+                    <%= index + 1 %>. <%= answer.body %>
```

Comment:
this page seems to have missed the marks as to what it is supposed to be reporting.  Rather than returning all the questions and answers, this is supposed to show:

1. Total number of users
2. Total number of questions
3. Total number of answers
4. Tenant API request counts for all Tenants (which I am unsure whether that means listing each tenants api requests or if it means the total api requests for all tenants (meaning just one number returned)